test-registry/function_configs_yaml/default_config.yaml

```diff
@@ -0,0 +1,5 @@
+name: default_config
```

Comment:
Let's add any default settings to a subfield in Config rather than create an additional worker-wide config file: https://github.com/open-lambda/open-lambda/blob/main/src/common/config.go#L19